WooCommerce/Classes/ViewRelated/Coupons/Add and Edit Coupons/AddEditCoupon.swift

```diff
@@ -0,0 +1,50 @@
+import SwiftUI
+import Yosemite
+
+struct AddEditCoupon: View {
```

Comment:
For this one and `CouponDetails`, do we have a convention when naming `SwiftUI` `View` s? Isn't it necessary to add the View prefix so It would be `AddEditCouponView` or for the file, AddEditCouponView.swift? It would look a bit clearer to me, if I see `CouponDetails` I tend to think that it is a model :)

Reply:
We still don't have a strict convention about the naming of the views in `SwiftUI`, in fact you will notice in the codebase some views with the `View` suffix and other views without the suffix. 
If we should follow the general Apple approach, in theory we should avoid the use of `*View` (see `VStack`, `Divider`, `Text`, `Spacer`, `Button`) for native SwiftUI views, otherwise if the view is a wrapper of an existing UIKit component, I believe it's ok to use the `*View` suffix. At the same time, Apple itself has some contrast on the naming convention, like `ScrollView`, `NavigationView`, etc...

Reply:
Great, thanks for the explanation :)